### DISCUSSION TITLE
libhogweed-6.dll and libgstcodecparsers-1.0-0.dll tagged as malware

### DISCUSSION THREAD

**Discussion by gustavmgr** [Xpra-org/xpra#3644] [General]
Hello there

I really want to use Xpra at work and we're pretty crazy about security so everything gets scanned. Apparently we're picking up something in these files:
" `libhogweed-6.dll` and `libgstcodecparsers-1.0-0.dll` have been flagged as a possible trojan by the `Vir.IT eXplorer AV engine`"

Does this look like something you recognise?

**Comment by totaam** [comment]
These two `DLL`s come directly from [MSYS2](https://www.msys2.org/) without being modified in any way, so the chance that they are malicious is vanishingly small.
False positives with virus scanners is something that happens regularly: https://github.com/Xpra-org/xpra/issues/3639#issuecomment-1271734294

I suggest that you just remove these two files, xpra should work OK without it, AFAICT this will only remove the ability to:
* use AES encryption as [libhogweed](https://packages.msys2.org/package/libhogweed?repo=msys&variant=x86_64) is used by the cryptography package
* use some audio codecs: [gst-plugins-bad-libs](https://packages.msys2.org/package/mingw-w64-x86_64-gst-plugins-bad-libs)

Or you could just use a different / better AV scanner, or report this as a false positive so they can fix it.
My guess is that if you don't report it, someone else will, as MSYS2 is used by a large number of open-source projects.

**Comment by gustavmgr** [comment]
Fantastic - I needed to know the origin so appreciate that very much
